scripts/alignment_score.py

```diff
@@ -0,0 +1,109 @@
+from re import I
+import sys
+import os
+
+# get indexes from file paths given a delimeter, default space
+def scrapeIndexes(path: str, delim: str = " ")-> list: 

```

Comment:
try to follow the style of the rest of the codebase, which is camel_case and no function annotations